From 3d862b968fb8a97285b7008ee9694cbb0f91523b Mon Sep 17 00:00:00 2001
From: Vishal Sagar <vishal.sagar@xilinx.com>
Date: Thu, 13 Dec 2018 16:10:07 +0530
Subject: [PATCH 0489/1851] dt-bindings: display: xlnx: mixer: Correct DRM
 fourcc for BGR8 and RGB8

commit c04a4ed1a5815d7fe839f2e7c8fecffab361ce47 from
https://github.com/Xilinx/linux-xlnx.git

Corrects the DRM fourcc code to be used for BGR8 and RGB8 color formats.

Signed-off-by: Vishal Sagar <vishal.sagar@xilinx.com>
Reviewed-by: Saurabh Sengar <saurabh.singh@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 Documentation/devicetree/bindings/display/xlnx/xlnx,mixer.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Documentation/devicetree/bindings/display/xlnx/xlnx,mixer.txt b/Documentation/devicetree/bindings/display/xlnx/xlnx,mixer.txt
index 0ed4705c7b4e..635576bf0a46 100644
--- a/Documentation/devicetree/bindings/display/xlnx/xlnx,mixer.txt
+++ b/Documentation/devicetree/bindings/display/xlnx/xlnx,mixer.txt
@@ -45,8 +45,8 @@ Optional properties:
 
 Supported Formats:
 	Mixer IP Format		Driver supported Format String
-	 BGR888			"BG24"
-	 RGB888			"RG24"
+	 BGR888			"RG24"
+	 RGB888			"BG24"
 	 XBGR2101010		"XB30"
 	 XRGB8888		"XR24"
 	 RGBA8888		"RA24"
-- 
2.31.1

